package.json

```
@@ -20,6 +20,7 @@
     "binary-release-tag": "b5.0.1",
     "binary-release-name": "5.0.1"
   },
+  "binary": "ffmpeg",
```

Let's create two packages in this repo (`ffmpeg-static` & `ffprobe-static`) whose `index.js`, `install.js` & `package.json` is just copied from here (the root directory), except that
- *this* `package.json` here would just have a dummy `binary` field (e.g. `"<overriden-by-packages>"`) and *theirs* (`{ffmpeg,ffprobe}-static/package.json`) would contain `"ffmpeg"` & `"ffprobe"`, respectively;
- *their* `package.json`s would have a different `name`.

In the root directory here, we would effectively have a template for the two packages `{ffmpeg,ffprobe}-static` that get published.

```suggestion
  "binary": "<overriden-by-packages>",
```